site/styles.css

```diff
@@ -1,156 +1,242 @@
+:root {
+  /* Mute (black/navy) */
+  --mute-darker: #020202;
+  --mute-dark: #121214;
+  --mute: #18191c;
+  --mute-light: #27292f;
+
+  /* Accent (blue) */
+  --accent-light: #8cb0de;
+  --accent: #4375b5;
+
+  /* Text */
+  --text-black: #151314;
+  --text-red: #c54445;
+  --text-grey: #737f8d;
+  --text-white: white; }
+
```

Comment:
:+1: for CSS variables.

I totally disagree with this brace style (it's not 1TBS :D), But Whatever.

Reply:
![image](https://user-images.githubusercontent.com/9429556/33525740-cf4651b8-d82d-11e7-85ef-0c5c53a4466f.png)